src/constants/constants.ts

```diff
@@ -52,13 +60,21 @@ export const METHODS: Record<Method, MethodData> = {
     returnEncoding: Encoding.BYTES,
   },
   [Method.GET_DATA]: {
-    // https://github.com/ERC725Alliance/erc725/blob/main/docs/ERC-725.md#erc725y
+    // https://github.com/ERC725Alliance/ERC725/blob/v4.0.0/docs/ERC-725.md#erc725y
     sig: '0x4e3e6e9c',
     gas: numberToHex(2000000),
     gasPrice: numberToHex(100000000),
     value: numberToHex(0),
     returnEncoding: Encoding.BYTES_ARRAY,
   },
+  [Method.GET_DATA_BATCH]: {
+    // https://github.com/ERC725Alliance/ERC725/blob/v5.1.0/docs/ERC-725.md#erc725y
+    sig: '0xdedff9c6',
+    gas: numberToHex(2000000),
+    gasPrice: numberToHex(100000000),
```

Comment:
I think we will need to make it dynamic at some point no? cc @CallumGrindle @magalimorin18

Reply:
whats this being used for? The lib doesn't send any transactions does it?